src/irc/core/channels-query.c

```diff
@@ -50,22 +50,7 @@
 #include <irssi/src/irc/core/irc-channels.h>
 #include <irssi/src/irc/core/servers-redirect.h>
 
-enum {
-	CHANNEL_QUERY_MODE,
-	CHANNEL_QUERY_WHO,
-	CHANNEL_QUERY_BMODE,
-
-	CHANNEL_QUERIES
-};
-
-#define CHANNEL_IS_MODE_QUERY(a) ((a) != CHANNEL_QUERY_WHO)
-
-typedef struct {
-	int current_query_type; /* query type that is currently being asked */
-        GSList *current_queries; /* All channels that are currently being queried */
-
-	GSList *queries[CHANNEL_QUERIES]; /* All queries that need to be asked from server */
-} SERVER_QUERY_REC;
+#define WHO_Ps_PPtna_745 "WHO %s %%tna,745"
```

Comment:
what even is this name

Reply:
the name is a direct copy of the string content... can you suggest a good name? The reason I have this  as a constant is that the similar string is used in more than 1 place and this way when editing the code later on I will be sure to notice that the string must be changed in more than 1 place

Reply:
oooh lol, P means percentage, now i get it. But okay, just add a comment next to it explaining what it means? WHOX is cryptic

Reply:
now resolved by your suggestion below